src/WebSite/Bit.Platform.WebSite/Web/Components/Footer.razor

```diff
@@ -1,3 +1,35 @@
-﻿<footer class="footer">
-    <div>© 2022 Bit team. All rights reserved.</div>
+﻿<footer class="footer-container">
+    <section class="footer-section divider">
+        <div class="bit-column">
+            <div class="bit-icon" />
+            <div class="line">Whoooo we are and what we do in bit or why we want to resolve others issus</div>
```

Comment:
is this a typo?
`Whoooo`

Reply:
so when are we going to decide about the final tests?
@maryamhdr

Reply:
@ysmoradi when the final content of the `BitPlatform` website is gonna be ready?